warehouse/templates/includes/packaging/project-data.html

```diff
@@ -199,7 +199,20 @@ <h6>{% trans %}Project links{% endtrans %}</h6>
 <div class="sidebar-section unverified">
   <h6>{% trans %}Meta{% endtrans %}</h6>
   <ul>
-  {% if license %}
+  {% if release.license_expression %}
+    <li>
+      <span>
+        <strong>{% trans %}License Expression:{% endtrans %}</strong> {{ release.license_expression }}
```

Comment:
I would use just "License" for the user visible string here.

Reply:
I'm waiting for a more thorough review from @pypi/warehouse-committers before I address suggestions/questions.

Reply:
Agreed, we should distinguish between the old "License" and the new "License-Expression" here.